docs/helm-deployment-acs-with-ai.md

```diff
@@ -50,4 +33,4 @@ export ALF_DB_PWD="<db_password>"
 export QUAY_PULL_SECRET="<quay_pull_secret_name>"
 ```
 * For additional information on the S3 Connector benefits, installation and configuration, see the [Alfresco Content Connector for AWS S3 documentation](https://docs.alfresco.com/s3connector/concepts/s3-contentstore-overview.html).
-* For additional information on the `AI_AWS_*` variables and configuration, see the [AWS Setup documentation](https://github.com/Alfresco/alfresco-ai-transformers/blob/master/AWS_Setup.md).
\ No newline at end of file
+* For additional information on the `AI_AWS_*` variables and configuration, see the [AWS Setup documentation](https://docstest.alfresco.com/intelligence/concepts/aws-setup.html).
```

Comment:
Minor: Change to docs.alfresco.com now ? Otherwise, we need to remember to come back later. Either way, external users will see a broken link until the docs are published later this week.